### PR TITLE
Fixed a regression issue with load_all() function

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/commons#655
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/commons/objects/__init__.py [hunk 1]
```diff
@@ -201,7 +202,9 @@ def load(self):
                 pass
 
         _copy = self._copy_vars()
-
+        # Check if self.value already set, use it
+        if self.value.find('{') < 0:
```
Comment (r0h4n) [inline tendrl/commons/objects/__init__.py]: self.render is already called on L167. It safe to assume self.value is rendered correctly, no need to check

Comment (shtripat) [inline tendrl/commons/objects/__init__.py]: `_copy_vars()` sets the old `value` field again in `_copy`. So I check if self.value doesn't contain `{` it means its properly rendered and so I assign it back to `_copy` variable's `value` field.

Comment (r0h4n) [inline tendrl/commons/objects/__init__.py]: Well you can include the rendered self.value in _copy_vars method, but dont do this self.value.find

Comment (shtripat) [inline tendrl/commons/objects/__init__.py]: Including `value` field in `_copy_vars()` causes an error `TypeError: __init__() got an unexpected keyword argument 'value'` while normal `load()` calls for entities, so took this path :) Just changed `_copy_vars` and saw this error again.

Comment (shtripat) [inline tendrl/commons/objects/__init__.py]: May be we can copy `self.value` un-conditionally to `_copy`

**Pull request merged**